From: Modestas Vainius <modax@debian.org>
Subject: Fix mysql_config call for Debian MySQL
Last-Update: 2012-06-02

* As of 5.5.23, mysql_config_pic is shipped inside libmysqld-pic package.
--- a/cmake/modules/FindMySQLAmarok.cmake
+++ b/cmake/modules/FindMySQLAmarok.cmake
@@ -13,7 +13,7 @@
 # For details see the accompanying COPYING-CMAKE-SCRIPTS file.
 
 if(NOT WIN32)
-    find_program(MYSQLCONFIG_EXECUTABLE NAMES mysql_config mysql_config5 HINTS ${BIN_INSTALL_DIR})
+    find_program(MYSQLCONFIG_EXECUTABLE NAMES mysql_config_pic mysql_config mysql_config5 HINTS ${BIN_INSTALL_DIR})
 endif(NOT WIN32)
 
 find_path(MYSQL_INCLUDE_DIR mysql.h PATH_SUFFIXES mysql mysql5/mysql)
